src/core/Notification/InlineNotification/InlineNotification.md

```diff
@@ -0,0 +1,37 @@
+```js
+import { InlineNotification, Paragraph } from 'suomifi-ui-components';
+
+<>
+  <InlineNotification
+    closeText="Close"
+    labelText="Lorem ipsum dolor sit"
+  >
+    <Paragraph>
```

Comment:
If we conclude that we can't adjust the paragraph styles as they are handled now, these will have to change somehow as well. I think the best course of action might be to just leave the semantics and styling for the projects and mention preferred semantics in the documentation. Any thoughts?

Reply:
I using paragraph only in 1 example.